### DISCUSSION TITLE
Clio 5 easy link mysense active problem.

### DISCUSSION THREAD

**Discussion by batuhanckc5858** [cedricp/ddt4all#1331] [Q&A]
Hello, I activated the My Sense menus in the Clio 5 vehicle. But when I change the driving modes, there is no change in the throttle response and steering response. Previously, when I pressed the eco button, the throttle response decreased, now it does not decrease. Can you help me with this? Is there a setting I missed?

**Comment by x5nder** [comment]
> Hello, I activated the My Sense menus in the Clio 5 vehicle. But when I change the driving modes, there is no change in the throttle response and steering response. Previously, when I pressed the eco button, the throttle response decreased, now it does not decrease. Can you help me with this? Is there a setting I missed?

I'm looking to do the same, can you tell me exactly which settings you've changed (ideally before/after)? I'll do my best to figure out what's happening.
